lib/Mojolicious/Plugin/AssetPack/Pipe/Export.pm

```diff
@@ -0,0 +1,92 @@
+
+package Mojolicious::Plugin::AssetPack::Pipe::Export;
+
+use Mojo::Base 'Mojolicious::Plugin::AssetPack::Pipe';
+use Mojo::Util 'spurt';
+
+use File::Basename 'dirname';
+use File::Path 'make_path';
+use File::Spec;
+
+has 'export_dir';
+has 'use_checksum_subdir' => 1;
+
+sub process {
+	my ($self, $assets) = @_;
+
+	my $dir = $self->export_dir // $ENV{'MOJO_ASSETPACK_EXPORT_DIR'};
+	unless ( defined $dir and -e -w -d $dir ) {
+		die "Missing or inaccesable export base directory '$dir'" 
+	}
+
+	$assets->each(sub {
+		my $asset = shift;
+
+		my $path;
+		if ( $self->use_checksum_subdir ) {
+			my $file = sprintf( "%s.%s", $asset->name, $asset->format );
+			$path = File::Spec->catfile( $dir, $asset->checksum, $file );
+		}
+		else {
+			my $file = sprintf( "%s-%s.%s", $asset->name, $asset->checksum, $asset->format );
+			$path = File::Spec->catfile( $dir, $file );
+		}
+		make_path dirname($path) unless -d dirname($path);
+		spurt $asset->content => $path;
+	});
+}
+
+1;
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Plugin::AssetPack::Pipe::Export - Export processed assets to directory
+
+=head1 SYNOPSIS
+
+  use Mojolicious::Lite;
+  
+  # "Export" comes last!
+  plugin AssetPack => {pipes => [qw(... Export)]};
+  app->asset->pipe('Export')->export_dir("/some/path/in/webroot");
+  # app->asset->pipe('Export')->use_checksum_subdir(0);
+
+=head1 DESCRIPTION
+
+L<Mojolicious::Plugin::AssetPack::Pipe::Export> will export the processed assets
+to the given directory so you can have them served directly by your webserver.
+
+=head1 ATTRIBUTES
+
+=head2 export_dir
```

Comment:
Consistency is king: Both "synopsis" and a good description is required. I disagree that it's pointless. I find it very helpful to see an example value passed in so I can tell the type without reading a long description.